Market Maker Guide.md

```diff
@@ -330,22 +330,14 @@ Balance:
 Please report issues/bugs by running `report` from within `xud ctl`.
 
 # Tips 'n Tricks
-* Raiden currently requires direct channels with trading partners. We have a temporary check in place, that discards raiden-related orders (all pairs which include WETH, DAI...), if `xud` can't find a direct channel to the trading partner. You can switch this check off by setting `raidenDirectChannelChecks=false` in your `xud.conf`. Before you do that, read [this explainer of the issue](https://github.com/ExchangeUnion/xud/issues/1068).
-* Permanently set the alias `xud` to launch `xud ctl` from anywhere:
-Add the line `alias xud="bash ~/xud.sh"` to the end of `~/.bashrc` or `~/.bash_aliases` on Linux and `bash_profile` on Mac, then `source` the file.
-* `xud ctl` allows to use an L1/L2 client's cli:
-```bash
-bitcoin-cli --help
-litecoin-cli --help
-geth --help
-lndbtc-lncli --help
-lndltc-lncli --help
-raiden --help
-xucli --help
-```
-* To inspect logs:
+* If you are syncing the full setup and `geth` is **stuck at 99.99%** syncing status and can't catch up to 100% within 24 hours after reaching 99.99%, you are likely running geth on a HDD or any other drive that is too slow for geth to catch up with the chain. The only advise we have here is: switch to a faster SSD (best 150 MB/s or higher)
```

Comment:
Mainnet takes longer than 24 hours to get to 100%. Just for reference my machine with a SATA SSD, 8 gb of ram, a suitable cpu and 100 mbit uplink took like three days

Reply:
True, my machine (allocated geth 8192 MB cache) took a bit less than 24h to get to 99.99% on mainnet. Since it highly depends on cache setting/RAM/resources on how fast the replaying states takes to catch up to 100% (`Ready`), I'd simply refer to do a read/write speed test if not catching up within 3 days.